src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -65,7 +65,9 @@ endif()
 file(GLOB_RECURSE SOURCES core/*.cpp graph/*.cpp frontend/*.cpp utils/*cpp pyopenvino.cpp)
 list(FILTER SOURCES EXCLUDE REGEX frontend/onnx|tensorflow|paddle/* )
 
-pybind11_add_module(${PROJECT_NAME} MODULE NO_EXTRAS ${SOURCES})
+set(CHANGED_PROJECT_NAME _${PROJECT_NAME})
```

Comment:
Would it be possible to modify `PROJECT_NAME` value instead of introducing a new `CHANGED_PROJECT_NAME` variable?

Reply:
`PROJECT_NAME` is also used here: https://github.com/openvinotoolkit/openvino/pull/13427/files#diff-23eb9571d192e51e931e95391f215d0a49cad8e229f095bc26e5230fad4ba571L110

Reply:
Maybe we can stick to `pyopenvino` in CMake files? I do not actually see any benefit of such change as adding `_` is more of the Python thing that will suggest users that module is private. CMake "as-is" is our building system that does not take privacy of module into the account, it simply does not matter here.

However, if using `_pyopenvino` once as destination name, it should be used everywhere else as well. @ilyachur what do you think about it, what are the CMake's guidelines in this situation?

Reply:
@jiwaszki we also need to have `_pyopenvino` in CMake as there are lines of code which copies lib, which is renamed, and other files to install dir.

Reply:
@jiwaszki ok, found the way how to reduce changes in CMake files: keep `pyopenvino` as target name :)